py/desispec/resolution.py

```diff
@@ -94,4 +105,32 @@ def to_fits_array(self):
         """
         return self.data
 
-#- (Unit tests moved to desispec.test.test_resolution)
\ No newline at end of file
+def _gauss_pix(x, mean=0.0, sigma=1.0):
+    """
+    Utility function to integrate Gaussian density within pixels
+
+    Args:
+        x (1D array): pixel centers
```

Comment:
Good idea, though this would require adding an optional argument to the Resolution constructor to specify what those bin centers/edges are.  If we do this, I might favor switching the sigma units to be Angstroms (currently pixels) and specify the wavelength bins in Angstroms.  I'll think about how to express this cleanly without complicating the interface.  "if `data` is 1D it is interpreted as sigmas and then the `wavelength` option is required, otherwise it is ignored" sounds a bit messy.  But it would be good to support this somehow.